src/common/HTTPRequest.ts

```diff
@@ -150,6 +166,126 @@ export class HTTPRequest {
     return this._url;
   }
 
+  hasCooperativeInterceptHandlers(): boolean {
+    return this._pendingCooperativeInterceptionHandlers.length > 0;
+  }
+
+  setAllowCooperativeRequestInterceptionMode(isEnabled: boolean): boolean {
```

Comment:
Is there a specific reason for returning the boolean again? Otherwise, I'd change this to void.